components/inputs/docs/input-date-time.md

```diff
@@ -327,16 +275,14 @@ Note: All `*value` properties should be in ISO 8601 combined date and time forma
 * `inline-help`: Help text that will appear below the input. Use this only when other helpful cues are not sufficient, such as a carefully-worded label.
 <!-- docs: end hidden content -->
 
-### Accessibility Properties
+## Accessibility
 
-To make your usage of `d2l-input-date-time-range` accessible, use the following properties when applicable:
+The date and time components generally follow the W3C's best practice recommendations for a [Date picker dialog](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/examples/datepicker-dialog/). For details on the accessibility of the calendar within the date input components, see [Calendar Accessibility](../calendar#accessibility). A few notable accessibility-related features of these components are:
 
-| Attribute | Description |
-|--|--|
-| `label` | **REQUIRED** [Acts as a primary label on the input](https://www.w3.org/WAI/tutorials/forms/labels/). Visible unless `label-hidden` is also used. |
-| `label-hidden` | Use if label should be visually hidden but available for screen reader users |
-| `end-label` | Accessible label for the second date-time input |
-| `start-label` | Accessible label for the first date-time input |
+* Explicitly stating the expected date format in both the description tooltip and the placeholder text
+* Trapping focus in the date input dropdown
+* When the date input is opened with the keyboard, focus goes to either selected date, today, or earliest valid date if today is prior to `min-date`
```

Comment:
Let me know thoughts on these and if there are other ones we would want to include

Reply:
The only thing I can think of is to maybe explain why for some of these points? Or at least for the second one, do we want to explain why exactly we're trapping the focus?

Reply:
I think it would be better to phrase these in active tense (I think that's what it's called?) so that instead of "Trapping focus in the ..." it would be "Focus is trapped in the ..."

And "Explicitly stating the ..." could be "The expected date format is indicated both in placeholder text and a tooltip available on hover and focus."

( Incidentally: I think we should improve how we offer the date format, since the tooltip is hard to trigger in some scenarios. I'll create a Jira ticket )